scenes/components/multiplayer_connection/multiplayer_connection.gd

```diff
@@ -0,0 +1,79 @@
+extends Node
+
+signal client_connected(peer_id: int)
```

Comment:
Remember to document stuff. Also I'd reorder these variables in accordance to the styleguide.